NPE in QueryMapFile

### Describe your problem!

reported by Play Console:

```
java.lang.RuntimeException: 
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3732)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:3906)
  at android.app.servertransaction.LaunchActivityItem.execute (LaunchActivityItem.java:103)
  at android.app.servertransaction.TransactionExecutor.executeCallbacks (TransactionExecutor.java:135)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:95)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2259)
  at android.os.Handler.dispatchMessage (Handler.java:106)
  at android.os.Looper.loopOnce (Looper.java:210)
  at android.os.Looper.loop (Looper.java:299)
  at android.app.ActivityThread.main (ActivityThread.java:8105)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:556)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1045)
Caused by: java.lang.NullPointerException: 
  at cgeo.geocaching.helper.QueryMapFile.onCreate (QueryMapFile.java:27)
  at android.app.Activity.performCreate (Activity.java:8238)
  at android.app.Activity.performCreate (Activity.java:8206)
  at android.app.Instrumentation.callActivityOnCreate (Instrumentation.java:1329)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3705)
```


### How to reproduce?

unclear

### Actual result after these steps?

_No response_

### Expected result after these steps?

_No response_

### Reproducible

Unclear

### c:geo Version

2022.06.06

### System information

_No response_

### Additional Information

115 occurrences so far

--- a/main/src/cgeo/geocaching/helper/QueryMapFile.java
+++ b/main/src/cgeo/geocaching/helper/QueryMapFile.java
@@ -24,7 +24,12 @@ public class QueryMapFile extends AppCompatActivity {
         final Bundle bundle = getIntent().getExtras();
         final boolean forceAndFeedback = null != bundle && bundle.getBoolean(getString(R.string.cgeo_queryMapFile_actionParam));
 
-        final String mapFile = PersistableFolder.OFFLINE_MAPS.getUri().toString();
+        String mapFile;
+        try {
+            mapFile = PersistableFolder.OFFLINE_MAPS.getUri().toString();
+        } catch (NullPointerException e) {
+            mapFile = null;
+        }
         if (forceAndFeedback || StringUtils.isNotEmpty(mapFile)) {
             try {
                 final Intent intent = new Intent(Intent.ACTION_SENDTO);

